Tools/CISupport/build-webkit-org/config.json

```diff
@@ -230,7 +230,7 @@
                   },
                   { "name": "Apple-Tahoe-Release-WK2-Site-Isolation-Tree-Tests", "factory": "TestLayoutFactory",
```

Comment:
I think you'll need to change the factory to `TestLayoutAndAPIOnlyFactory` if you want to add API tests.